src/acquisition/fluview/fluview_update.py

```diff
@@ -132,23 +179,201 @@ def get_ilinet_data(row):
     'n_patients': optional_int(row[14]),
   }
 
+def get_clinical_data(row):
+  if row[0] == 'REGION TYPE' and row != [
+    'REGION TYPE', 
+    'REGION', 
+    'YEAR', 
+    'WEEK', 
+    'TOTAL SPECIMENS', 
+    'TOTAL A',
+    'TOTAL B', 
+    'PERCENT POSITIVE', 
+    'PERCENT A', 
+    'PERCENT B'
+  ]:
+    raise Exception('header row has changed for clinical lab data.')
+  if len(row) == 1 or row[0] == 'REGION TYPE':
+    # this is a header row
+    return None
+  if row[4] == 'X':
+    # data is not reported, ignore this row
+    return None
+  # ignore percentage calculations for now
+  return {
+    'location': fluview_locations.get_location_name(*row[:2]),
+    'epiweek': join_epiweek(int(row[2]), int(row[3])),
+    'total_specimens': int(row[4]),
+    'total_a': optional_int(row[5]),
+    'total_b': optional_int(row[6])
+  }
 
-def load_zipped_csv(filename):
-  """Read rows from a zipped CSV, which is expected to be named ILINet.csv."""
+def get_public_data(row):
+  if row[0] == 'REGION TYPE' and row != [
+    'REGION TYPE', 
+    'REGION', 
+    'SEASON_DESCRIPTION', 
+    'TOTAL SPECIMENS',
+    'A (2009 H1N1)', 
+    'A (H3)', 
+    'A (Subtyping not Performed)', 
+    'B', 
+    'BVic',
+    'BYam', 
+    'H3N2v'
+  ]:
+    raise Exception('header row has changed for public health lab data.')
+  if len(row) == 1 or row[0] == 'REGION TYPE':
+    # header row
+    return None
+  if row[3] == 'X':
+    # data is not reported, ignore this row
+    return None 
+  return {
+    'location': fluview_locations.get_location_name(*row[:2]),
+    'season': row[2],
+    'total_specimens': int(row[3]),
+    'total_a_h1n1': optional_int(row[4]),
+    'total_a_h3': optional_int(row[5]),
+    'total_a_no_sub': optional_int(row[6]),
+    'total_b': optional_int(row[7]),
+    'total_b_vic': optional_int(row[8]),
+    'total_b_yam': optional_int(row[9]),
+    'total_h3n2v': optional_int(row[10])
+  }
+
+def load_zipped_csv(filename, sheetname='ILINet.csv'):
+  """Read rows from a zipped CSV, which is expected to be named as specified
+  by the sheetname parameter. Default is ILINet.csv, for the default flu data."""
   with zipfile.ZipFile(filename) as f:
-    with f.open('ILINet.csv') as ff:
+    with f.open(sheetname) as ff:
       return [row for row in csv.reader(io.StringIO(str(ff.read(), 'utf-8')))]
 
-
-def get_rows(cnx):
+def get_rows(cnx, table='fluview'):
   """Count and return the number of rows in the `fluview` table."""
```

Comment:
Small request: please update doc to say something to the effect of "number of rows in the given table, `fluview` by default"